_episodes_rmd/04-ggplot2.Rmd

```diff
@@ -62,20 +62,25 @@ this fashion allows for extensive flexibility and customization of plots.
 To build a ggplot, we will use the following basic template that can be used for different types of plots:
 
 ```
-ggplot(data = <DATA>, mapping = aes(<MAPPINGS>)) +  <GEOM_FUNCTION>()
+<DATA> %>%
+    ggplot(aes(<MAPPINGS>)) +
+    <GEOM_FUNCTION>()
 ```
+Remember from the last lesson that the pipe operator `%>%` places the result of the left-hand side into the first argument of the right-hand side. **`ggplot2`** is a function that expects a data source to be the first argument, allowing us to pipe the data into the ggplot function.
 
 - use the `ggplot()` function and bind the plot to a specific data frame using
   the `data` argument
```

Comment:
Explanatory text needs updating; piping doesn't explicitly use the `data` argument

Reply:
What do you suggest to improve this? removing the "using the `data` argument" on the next line of code (71/72) or 
introducing the . explicitly calling `data = .` ? 

Do you have any other ideas or solutions?

Reply:
Remove the reference to the 'data' argument. Or maybe tailor the language to the use of the pipe, eg, "pipe a data frame into the `ggplot()` function to begin plotting" or something along those lines?

Reply:
Got it!

Reply:
How about specifying that, because `ggplot` uses the data as the first argument: 

*This allows for us to change from specifying the `data =` argument and pipe the data into a ggplot instead.*